cli/README.md

```diff
@@ -133,7 +157,8 @@ Loads the provided URLs in a headless browser several times to measure median We
 * `--url` (`-u`): A URL to benchmark.
 * `--number` (`-n`): Total number of requests to send.
 * `--file` (`-f`): File with URLs (one URL per line) to run benchmark tests for.
-* `--output` (`-o`): The output format.
+* `--output` (`-o`): The output format: Either "table" or "csv".
```

Comment:
@felixarntz same comment here WRT `--output` as opposed to `--format`

Reply:
@10upsimon See my reply above.